src/test/kotlin/it/pagopa/wallet/controllers/WalletControllerTest.kt

```diff
@@ -239,6 +239,25 @@ class WalletControllerTest {
             .json(jsonToTest)
     }
 
+    @Test
+    fun testGetWalletAuthDataSuccess() = runTest {
```

Comment:
You can add a test case on the WalletNotFoundException

Reply:
correct :-) done